src/marqo/tensor_search/enums.py

```diff
@@ -58,6 +58,7 @@ class EnvVars:
     MARQO_MAX_DELETE_DOCS_COUNT = "MARQO_MAX_DELETE_DOCS_COUNT"
     MARQO_MAX_NUMBER_OF_REPLICAS = "MARQO_MAX_NUMBER_OF_REPLICAS"
     MARQO_DEFAULT_EF_SEARCH = "MARQO_DEFAULT_EF_SEARCH"
+    MARQO_MAX_ADD_DOCS_COUNT = "MARQO_MAX_ADD_DOCS_COUNT"
```

Comment:
We don't have this in v2. The limit is currently hard-coded

Reply:
removed